src/app/components/Modals/UnbondErrorModal.tsx

```diff
@@ -0,0 +1,40 @@
+import {
+  Button,
+  Dialog,
+  DialogBody,
+  DialogFooter,
+  Heading,
+  Text,
+} from "@babylonlabs-io/bbn-core-ui";
+import Image from "next/image";
+
+import warningIcon from "@/app/assets/warning-icon.svg";
+
+interface UnbondErrorModal {
+  isOpen: boolean;
```

Comment:
In #373 we use `open`
Here we use `isOpen`. I suggest choosing one approach and stick with it